### ISSUE TITLE
Fail to calculate fan_in and fan_out

### ISSUE THREAD

**Issue by ishepard** [terryyin/lizard#250]
I am having problems in calculating `fan_in` and `fan_out` of the methods. I tried different languages, different approaches (analyze recursively a directory, analyze only one file, etc.), but the result is always the same: `fan_in` and `fan_out` = 0.

What am I doing wrong? 

This is the code I am using:
```
import lizard

an = lizard.analyze_file("/path/to/HTTP.c")
# also tried analyze, analyze_file and giving the directory

print(an.filename)
for func in an.function_list:
    print(func.fan_in)
    print(func.fan_out)
```
the output is always 0.

**Comment by terryyin** [comment]
The fan_in/out feature is an extension of Lizard, and I think at this moment it’s not working.

**Comment by hgn** [comment]
Yes, it is not working yet. I realized that too. ;-) Beside this: thank you @terryyin for lizard and the effort in developing that tool!
